### DISCUSSION TITLE
Payment methods with php8

### DISCUSSION THREAD

**Discussion by jlores** [PrestaShop/PrestaShop#32354] [Q&A]
Hello,

Do you know if there is a payment module that is compatible with php8?

I have tried with the latest Paypal module and with Prestashop checkout and they do not work with php8 or php 8.1

Best regards

**Comment by kpodemski** [comment]
I'll ping somebody from the checkout team because I'm pretty sure it should be compatible. 

cc @Matt75

**Comment by jlores** [comment]
I have tried many times, but with 8.0 and 8.1 it stops in the means of payment and they do not appear

Many thanks

**Comment by jlores** [comment]
It's weird, because no one complains that this doesn't work with php8.x but everyone reports that it installs with php7.x

More than anything, because php8.x is the main improvement in prestashop 8
